crypto/encrypt.go

```diff
@@ -11,3 +17,50 @@ func Encrypt(pubKey *btcec.PublicKey, data []byte) ([]byte, error) {
 func Decrypt(privKey *btcec.PrivateKey, data []byte) ([]byte, error) {
 	return btcec.Decrypt(privKey, data)
 }
+
+func EncryptWithAES256(key, nonce, data []byte) ([]byte, error) {
```

Comment:
```suggestion
func EncryptWithAES256(secretKey, nonce, data []byte) ([]byte, error) {
```
I suggested to rename the `key` to `secretKey`.

Reply:
Ah yes, I agree